### ISSUE TITLE
question about input of attention in the file seq2seq_with_attn.py for ch5

### ISSUE THREAD

**Issue by shihkung** [PacktPublishing/Deep-Learning-with-TensorFlow-and-Keras-3rd-edition#2]
for the call function in the Decoder class, 
the input to the self.attention should be (state, encoder_out)?

def call(self, x, state, encoder_out):
        x = self.embedding(x)
        context, alignment = self.attention(x, encoder_out)
        x = tf.expand_dims(
                tf.concat([
                    x, tf.squeeze(context, axis=1)
                ], axis=1), 
            axis=1)
        x, state = self.rnn(x, state)
        x = self.Wc(x)
        x = self.Ws(x)
        return x, state, alignment
